Sources/Paging/PageEvent.swift

```diff
@@ -35,6 +33,73 @@ open class PageEvent<T: Any> {
         
         var mediatorLoadStates: LoadStates? = nil
         
+        private func mapPages<R: Any>(_ transform: @escaping (TransformablePage<T>) -> TransformablePage<R>) -> Insert<R> {
```

Comment:
To improve code readability, what do you think about changing 'R' to 'Result'?

Reply:
No, R means Result

Reply:
No, R means Result

Reply:
OK, Got it, Do what you want